### PR TITLE
Footer improvements

### LINKED ISSUE #120: Improve footer
1. Links overlap on each other.
2. "Go to top" button has icon which not aligned to center.

footer

### PR DESCRIPTION
**Pull request by psmyrek**
### Related Issues

Closes #120 

### Short Description and Why It's Useful

It fixes 2 issues:
1) Footer width is now the same as rest of the page content. Also link texts have been adapted to fit inside columns. The last column with social icons has width automatically set basing on its content to properly align with the right side of page content. Also icon margin is added to all these icons except the last one.
2) Arrow icon is now positioned in the center of "Back to top" button. Current solution works for any icon size.

### Screenshots of Visual Changes before/after (If There Are Any)

Footer

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: components/atoms/a-back-to-top.vue [hunk 1]
```diff
@@ -11,7 +11,6 @@
         icon="chevron_down"
         icon-size="20px"
         icon-color="white"
-        view-box="0 -4 14 14"
```
Comment (psmyrek) [inline components/atoms/a-back-to-top.vue]: I removed `view-box` because it is not needed if Storefront UI icon is used.

Comment (qiqqq) [reply components/atoms/a-back-to-top.vue]: ok

File: components/organisms/o-footer.vue [hunk 2]
```diff
@@ -95,11 +95,11 @@ export default {
           name: 'About us',
           children: [
             {
-              name: 'About us (Magento CMS)',
+              name: 'About us',
```
Comment (qiqqq) [inline components/organisms/o-footer.vue]: Wiem, że to powinno być wyłapane w CR odnośnie footera, ale dopiero teraz zauważyłem. Name w tym obiekcie, trzeba wrzucić w funkcję do tłumaczeń

Comment (psmyrek) [reply components/organisms/o-footer.vue]: This goes already through translation function:

`:label="$t(link.name)"`

in this line https://github.com/DivanteLtd/vsf-capybara/pull/129/files#diff-a374d3c1a60eceac552624122a936901R12

Comment (qiqqq) [reply components/organisms/o-footer.vue]: Right, sorry :)

**Pull request merged**